### DISCUSSION TITLE
System Page - Sound Profile cannot be set

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#32] [Fixed Issues]
The Sound Profile option slider (on the System page) can be set but no access to the actual sound profiles is allowed. If I exit the System page and return, the Sound Profile option is disabled again.

Chris

rAudio 1 20201224 on RPi 3B+ with Merus Amp DAC.

**Comment by ceejayemm** [comment]
Just finished reporting this issue when update 20201228 became available. Applied this update and it seems to fix this problem and makes Sound Profiles available again but still have other issues see update to #26 

Chris

**Reply by Sagnasty** [reply]
I noticed that if I converted to rAudio 1 using a backup from the last RuneAudio+r e6 rev, the sound profile behavior was exactly as you described.

The only way I could get the sound profile option to perform correctly was to rebuild from image without restoring a backup (upgrading to 20201228 did not work in my case).

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+
